app/controllers/local_petitions_controller.rb

```diff
@@ -0,0 +1,30 @@
+require 'postcode_sanitizer'
+
+class LocalPetitionsController < ApplicationController
+  respond_to :html
+
+  def index
```

Comment:
Ah that's true, I was somehow thinking we are persisting also the constituency name to our db.